packages/lib-classifier/src/store/ClassificationStore.js

```diff
@@ -168,7 +169,7 @@ const ClassificationStore = types
         self.classificationQueue.add(classification)
       } else {
         try {
-          const response = yield client.post(`/${self.type}`, { classifications: classification })
+          const response = yield client.post(`/${self.type}`, { classifications: classification }, undefined, host)
```

Comment:
This saves all classifications as anonymous.

Reply:
Auth hasn't been hooked up in the classifier AFAIK

Reply:
They already are, if there was no authorisation argument being passed in